Neos.ContentRepository.Core/Classes/Projection/ContentGraph/Filter/FindBackReferencesFilter.php

```diff
@@ -21,34 +31,91 @@ final class FindBackReferencesFilter
      * @internal (the properties themselves are readonly; only the write-methods are API.
      */
     private function __construct(
+        public readonly ?NodeTypeConstraints $nodeTypeConstraints,
+        public readonly ?SearchTerm $nodeSearchTerm,
+        public readonly ?PropertyValueCriteriaInterface $nodePropertyValue,
+        public readonly ?SearchTerm $referenceSearchTerm,
+        public readonly ?PropertyValueCriteriaInterface $referencePropertyValue,
         public readonly ?ReferenceName $referenceName,
+        public readonly ?Ordering $ordering,
```

Comment:
The ordering is applied to the timestamp/property values of the target *node*..
We _could_ rename that to `$nodeOrdering`.
But adding an additional `$referenceOrdering` wouldn't be as easy because those don't have timestamp properties so we would have to add a complete new Value Object etc

Reply:
all other orderings are also named $ordering, so IMHO it is OK